src/dialects/oracle/data-types.js

```diff
@@ -302,6 +302,14 @@ module.exports = BaseTypes => {
     _getBindDef(oracledb) {
```

Comment:
Can you check for the value here and return DB_TYPE_VARCHAR if the type of value is bigint?